roles/grafana/tasks/provision-insolvency-dashboard.yml

```
@@ -0,0 +1,18 @@
+---
+
+- name: Provision insolvency dashboard
+  uri:
+    url: "{{ grafana_url }}/api/dashboards/db"
+    method: POST
+    url_username: "{{ item.grafana_admin_user }}"
+    url_password: "{{ item.grafana_admin_password }}"
+    body: "{{ lookup('template', 'roles/grafana/templates/insolvency_dashboard.json.j2') }}"
+    status_code: 200
+    body_format: json
+    validate_certs: no
+    force_basic_auth: yes
+    headers:
+      Content-Type: application/json
+  with_items:
+    "{{ kafka3_variables }}"
```

Make the variable expansion here a list item:

```suggestion
  with_items:
    - "{{ kafka3_variables }}"
```

Or collapse into a single line:

```suggestion
  with_items: "{{ kafka3_variables }}"
```